### DISCUSSION TITLE
How to use Inner Join in CreateQuery

### DISCUSSION THREAD

**Discussion by PabloRuiz-TT** [Azure/azure-mobile-apps#774] [Q&A] [answered]
Is there any way to create a query by doing inner join to my cloud tables and get the records to be included in my offline database.

**Comment by adrianhall** [comment, marked as answer]
Short version - no, there isn't a way to do an inner join easily.

Longer answer - sync the tables individually, then use store.ExecuteQueryAsync() to execute an explicit query.  The store is a SQLite database, so there will be differences, especially if you use complex data types (like DateTime, DateTimeOffset, Guid) instead of scalars.
